static/app/components/forms/controls/textarea.tsx

```diff
@@ -41,6 +41,14 @@ const TextArea = styled(TextAreaControl, {
 })`
   ${inputStyles};
   line-height: ${p => p.theme.text.lineHeightBody};
+
+  /** Allow react-autosize-textarea to freely control height based on props. */
+  ${p =>
+    p.autosize &&
+    `
+      height: auto;
+      min-height: auto;
```

Comment:
What about “unset”

Reply:
Hmm that works too

Reply:
https://github.com/getsentry/sentry/pull/38471/commits/3a7c6824163eb7bf32ac8eb86af78ca90136d94a